docs/readmes/howtos/call_tracing.md

```diff
@@ -0,0 +1,65 @@
+---
+id: call_tracing 
+title: Call Tracing
+hide_title: true
+---
+
+*Last Updated: 1/20/2021*
+
+# Call Tracing
+
+Magma supports basic traffic capture for troubleshooting purposes.
+Monitoring of control messaging flow and other traffic between the Magma access
+gateway and eNodeB devices is possible with this feature.
+
+As of the time of writing, we are in the process of adding filtering for
+specific protocols and allowing custom options through tshark.
+
+### Requirements
+
+Ensure you have the following:
+
+* a functional orc8r
+* a configured LTE network
+* a configured LTE gateway with eNodeB
+* subscribers that can attach to your LTE gateway
+* network running in un-federated mode
+
+### Initiating a Call Trace
+
+![Call Tracing Page](assets/nms/calltracing_page.png)
+
+The call tracing page can be accessed directly via the NMS sidebar.
+
+Click `Start New Trace`
+
+![Call Tracing Dialog](assets/nms/calltracing_dialog.png)
+
+`trace_id` must be a unique value among all the call traces on your network.
+
+You must specify a `gateway_id` to capture your call trace on. Call traces
+capturing across multiple gateways simultaneously are not supported at this
+time. If you wish to do so, start multiple call traces, one for each gateway.
+
+Once the call trace is started, you can stop and/or download the call trace
+on the same page. Under `Actions`, click the vertical ellipsis button for your
+call trace to see these options.
+
+### Viewing a Call Trace
+
+It is suggested that Wireshark is used to analyze call trace captures.
+
+![Call Tracing Actions](assets/nms/calltracing_actions.png)
+
```

Comment:
Can you add a debugging section mentioning how to debug issues if call tracing is not working as expected. Where to look, logs to collect etc.